packages/connect/src/data/deviceAuthenticityConfig.ts

```diff
@@ -119,11 +122,11 @@ export const deviceAuthenticityConfig: DeviceAuthenticityConfig = {
         },
     },
     T3W1: {
-        rootPubKeys: ['you shall not pass'], // TODO T3W1
-        caPubKeys: ['you shall not pass'], // TODO T3W1
+        rootPubKeys: ['you shall not pass'],
```

Comment:
@MiroslavProchazka it looks like you added here those TODOs but this file is updated semi-automatically so they get deleted. Shouldn't we create a issue or something to update the script that update this once it is necessary?